src/shared/containers/tag/index.js

```diff
@@ -1,17 +1,27 @@
 import Container from '../../components/container';
 import React, { Component } from 'react';
 import { fetchEvents } from '../../actions/events';
+import { fetchNews } from '../../actions/news';
 import Section from '../../components/section';
 import styles from './style.css';
 import fetch from '../../util/fetch-proxy';
 import { connect } from 'react-redux';
 import EventsList from '../../components/events-list';
+import NewsList from '../../components/news-list';
 
 export class Tag extends Component {
-  static fetchData = fetchEvents(fetch());
+  static fetchData = fetchNews(fetch());
+  static fetchEventsData = fetchEvents(fetch());
```

Comment:
This doesn't actually work - need to come up with a consolidated approach to fetch both events and news on the cold load.